docs/README.md

```diff
@@ -0,0 +1,137 @@
+|-------------------------------------|----------|---------|------------
+| letsencrypt_http_provider           | yes      |         | which http provider should be used: s3, local
+
+## Variables for dns-challenge
+
+| Variable                            | Required | Default | Description
+|-------------------------------------|----------|---------|------------
+| dns_user                            | yes      |         | username to access the DNS api
+| dns_password                        | yes      |         | password to access the DNS api
+| letsencrypt_dns_provider            | yes      |         | which DNS provider should be used: autodns, azure, hetzner, pebble, openstack
+
+## Global role variables
+
+| Variable                                 | Required | Default                              | Description
+|------------------------------------------|----------|--------------------------------------|------------
+| letsencrypt_conf_dir                     | no       | $HOME/letsencrypt                    | overwrite letsencrypt_conf_dir if you want to use another directory which is accessible for the user which runs the playbook
+| letsencrypt_prerequisites_packagemanager | no       | yum                                  | set the packagemanager which is used of the ansible_host. Possible values are all supported package managers from ansible package module
+| acme_staging_directory                   | no       | acme-staging-v02.api.letsencrypt.org | acme directory which will be used for certificate challenge
+| acme_live_directory                      | no       | acme-v02.api.letsencrypt.org         | acme directory which will be used for certificate challenge
+| account_key_path                         | no       | $letsencrypt_conf_dir                | path for account key of letsencrypt
+| csr_path                                 | no       | $letsencrypt_conf_dir/certs          | path for csr which is created for challenge
+| cert_path                                | no       | $letsencrypt_conf_dir/certs          | path for issued certificate
+| intermediate_path                        | no       | $letsencrypt_conf_dir/certs          | path for intermediate chain
+| fullchain_path                           | no       | $letsencrypt_conf_dir/certs          | path for full chain file (certificate + intermediate)
+| private_key_path                         | no       | $letsencrypt_conf_dir/certs          | path for private key
+| remaining_days                           | no       | 30                                   | min days remaining before certificate will be renewed
+| convert_cert_to                          | no       |                                      | format to convert the certificate to: `pfx`
+| validate_certs (bool)                    | no       |                                      | Gets used during integration tests with pebble server
```

Comment:
Why mention that this varible is a bool? We don't mention the datatype of all other variables.

Reply:
Removed this mention. Thought it would make sense here as most of the other vars are no bools or at least have defaults which tell if they are bools or not.